docs/known_issues.rst

```diff
@@ -300,42 +300,6 @@ If so, you can go ahead and try running ``setup.py`` again (in the new
 terminal).
 
 
-Creating a Time Object Fails with ValueError After Upgrading ``astropy``
```

Comment:
Why is this removal part of the PR?

Reply:
It is no longer relevant now that erfa is removed. I did consider trying to adjust the text, but then, given that this was for astropy 1.0, thought I might as well delete it. If there turns out to be a similar issue, we're better off adding a new one...